docs/user/nifi.rst

```diff
@@ -34,24 +34,44 @@ Install the Processors
 ~~~~~~~~~~~~~~~~~~~~~~
 
 To install the GeoMesa processors you will need to copy the nar files into the ``lib`` directory of your
-NiFi installation. There are currently three nar files:
+NiFi installation. There currently two common nar files, and seven datastore-specific nar files.
```

Comment:
Is it possible to combine these common nars into one?

Reply:
no, see https://nifi.apache.org/docs/nifi-docs/html/developer-guide.html#nars for details